transfer-data/introduction.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Introduction'
+description: 'This section explains how to set up a flow to transfer and transform data from an Axiom dataset to a destination.'
+---
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
+</Note>
+
+A flow is a way to transfer and transform data from an Axiom dataset to a destination that you choose. For example, you can use flows to retain the complete record of all events in Axiom while transferring high-value app events to a costly SIEM.
+
+The **Flows** tab allows you to move and transform data using APL and provides you with a unified experience that streamlines your data workflows.
+
+## Elements of a flow
+
+A flow consists of three elements:
+- A source. This is the Axiom dataset from which you transfer and transform data.
+- A transformation. This is the APL query you use to shape your data.
+- A destination. This is where you transfer the data.
+
+GIF
```

Comment:
TODO!

Reply:
Added the screen capture here: https://axiom-mano-flows.mintlify.app/process-data/introduction
Please let me know what you think.